services/QuillLMS/app/services/demo/create_admin_report.rb

```diff
@@ -4,24 +4,36 @@ class Demo::CreateAdminReport
 
   NUMBER_OF_CLASSROOMS_TO_DESTROY_SESSIONS_FOR = 20
   RANGE_OF_NUMBER_OF_SESSIONS_TO_DESTROY = 14..28 # 10-20% of 140
+  BATCH_DELAY = 1.minute
```

Comment:
Can you add a note in the PR description about the purpose of the BATCH_DELAY, in other words, why it is necessary?

Reply:
Explanation added! Thanks for flagging.

Reply:
Thank you.  That is super helpful.